utils/build.config

```diff
@@ -6,7 +6,7 @@ ARGOBOTS = v1.1
 PMDK = 1.12.1
 ISAL = v2.30.0
 ISAL_CRYPTO = v2.23.0
-SPDK = v22.01.1
+SPDK = v22.01.2
```

Comment:
Do we need to update RPM build spec for this change?

Reply:
yes: https://github.com/daos-stack/spdk/pull/52 https://github.com/daos-stack/dpdk/pull/24